cmux-tui/bindings/python/cmux/client_defaults.py

```
@@ -1,18 +1,64 @@
+            or ord(character) < 0x20
+            or 0x7F <= ord(character) <= 0x9F
+            or character in {"\u0085", "\u2028", "\u2029"}
+            or 0xD800 <= ord(character) <= 0xDFFF
+            for character in session
+        )
+    )
+    if invalid:
+        raise ValueError(
+            "session name must be a non-empty path component "
+            "without separators or control characters"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)